providers/ipstack.json

```diff
@@ -18,7 +18,8 @@
   "parameters": [
     {
       "name": "HTTP_SCHEME",
-      "description": "The scheme identifies the protocol to be used to access the IP Stack API. Allowed values are \"http\" or \"https\". Only \"http\" scheme is allowed in case of IP Stack FREE subscription."
+      "description": "The scheme identifies the protocol to be used to access the IP Stack API. Allowed values are \"http\" or \"https\". Only \"http\" scheme is allowed in case of IP Stack FREE subscription.",
+      "default": "https"
```

Comment:
I am torn here about having `https` as default. `http` is supported by free subscription but it's unsafe by default. On one hand, you need to manually set this if you have free subscription, on the other hand if this was `http` by default, it would be unsafe from the start and it's possible paid subscribers wouldn't change it anyway.

So okay, keep `https` _however_ make sure there is sufficiently informative message if I attempt to use `https` while on the free subscription.

Reply:
The error returned by OneSDK communicates clearly what is the problem when using `https` while on the free subscription. See attached screenshot.

<img width="662" alt="image" src="https://user-images.githubusercontent.com/5206165/157392887-7ebf415e-0b5e-496c-b8c9-9423bc24e9dd.png">